doc/man3/SSL_CTX_set0_CA_list.pod

```diff
@@ -17,10 +28,27 @@ SSL_get0_peer_CA_list - get or set CA list
  int SSL_CTX_add1_to_CA_list(SSL_CTX *ctx, const X509 *x);
  int SSL_add1_to_CA_list(SSL *ssl, const X509 *x);
 
+ void SSL_CTX_set_client_CA_list(SSL_CTX *ctx, STACK_OF(X509_NAME) *list);
+ void SSL_set_client_CA_list(SSL *s, STACK_OF(X509_NAME) *list);
+ STACK_OF(X509_NAME) *SSL_get_client_CA_list(const SSL *s);
+ STACK_OF(X509_NAME) *SSL_CTX_get_client_CA_list(const SSL_CTX *ctx);
+ int SSL_CTX_add_client_CA(SSL_CTX *ctx, X509 *cacert);
+ int SSL_add_client_CA(SSL *ssl, X509 *cacert);
+
  const STACK_OF(X509_NAME) *SSL_get0_peer_CA_list(const SSL *s);
 
 =head1 DESCRIPTION
 
+The following functions can be called on either the server or the client side.
```

Comment:
I think more clear to say `the below functions` rather than `the following functions`, and list them as soon as possible, so that the context is clear.

Reply:
I would also, as early as possible, discourage the use of the new functions to set a non-empty CA list in any client or dual-purpose client+server context.  The sending of client CA lists to the server is generally unnecessary, is a privacy leak and risks interoperability issues when the client HELLO gets too large.  Users should generally stick the older `_client_` functions, and perhaps those should even be described first?